website/docs/components/link/inline/partials/code/how-to-use.md

```diff
@@ -3,19 +3,19 @@
 The most basic invocation requires some content to be passed as children and either an `@href` or `@route` argument.
 
 ```handlebars
-<Hds::Link::Inline @href="...">Watch tutorial video</Hds::Link::Inline>
+Lorem <Hds::Link::Inline @href="...">ipsum dolor</Hds::Link::Inline> sit amet consectetur adipiscing elit.
 ```
 
 ### Color
 
 There are two available colors for a Inline Link: `primary` and `secondary`. The default is `primary`. To use a different color, set `@color` to `secondary`.
 
 ```handlebars
-<Hds::Link::Inline @color="primary" @href="...">Read tutorial</Hds::Link::Inline>
+Lorem <Hds::Link::Inline @color="primary" @href="...">ipsum dolor</Hds::Link::Inline> sit amet consectetur adipiscing elit.
```

Comment:
Since primary is the default, I wouldn't include a code example including it as a value. I would only include the code example with `@color="secondary"`.

Reply:
I would also delete `@color="primary"` from the other code examples.